src/client/datascience/kernel-launcher/kernelDaemonPreWarmer.ts

```diff
@@ -88,23 +82,23 @@ export class KernelDaemonPreWarmer {
     }
 
     // Handle opening of native documents
-    private async onDidOpenNotebookDocument(doc: NotebookDocument): Promise<void> {
-        // It could be anything, lets not make any assumptions.
-        if (isUntitledFile(doc.uri) || !isJupyterNotebook(doc)) {
-            return;
-        }
-        const kernelConnection = this.controllerManager.getSelectedNotebookController(doc)?.connection;
-        const isPythonKernel = kernelConnection ? isPythonKernelConnection(kernelConnection) : false;
-        const notebookMetadata = isPythonNotebook(getNotebookMetadata(doc));
-        if (
-            isPythonKernel ||
-            notebookMetadata ||
-            doc.getCells().some((cell: NotebookCell) => {
-                return cell.document.languageId === PYTHON_LANGUAGE;
-            })
-        ) {
-            await this.preWarmKernelDaemonPool();
-        }
+    private async onDidOpenNotebookDocument(_doc: NotebookDocument): Promise<void> {
+        // // It could be anything, lets not make any assumptions.
+        // if (isUntitledFile(doc.uri) || !isJupyterNotebook(doc)) {
+        //     return;
+        // }
+        // const kernelConnection = this.controllerManager.getSelectedNotebookController(doc)?.connection;
```

Comment:
This doesn't work, as in the controller manager, the document is opened and setting the preferred is done asynchronously. Also there's no such thing as a selected controller anymore.

I think we have an issue to track auto start of native notebook kernels, somethign to do with some UI (progress indicator) not appearing corrrectly.

either way, I've disabled this as it wont' work.